### ISSUE TITLE
equalToIgnoringWhiteSpace() and not(equalToIgnoringWhiteSpace()) should be supported

### ISSUE THREAD

**Issue by ChrisA89** [mkordas/hamcrest-to-assertj#1]
You've stated in the readme that you can not convert the following "due to AssertJ limitations"

> assertThat("string", equalToIgnoringWhiteSpace(" string"));
> assertThat("string", is(equalToIgnoringWhiteSpace(" string")));
> assertThat("string", not(equalToIgnoringWhiteSpace("tring")));
> assertThat("string", is(not(equalToIgnoringWhiteSpace("tring"))));

But these are supported. See [AbstractCharSequenceAssert.java#L797](https://github.com/joel-costigliola/assertj-core/blob/c41aad4d40fcdddda9b3b94803849730d128fe7d/src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java#L797) and [AbstractCharSequenceAssert.java#L829](https://github.com/joel-costigliola/assertj-core/blob/c41aad4d40fcdddda9b3b94803849730d128fe7d/src/main/java/org/assertj/core/api/AbstractCharSequenceAssert.java#L829)

**Comment by mkordas** [comment]
@ChrisA89 you are right! It was me who requested this in AssertJ in https://github.com/joel-costigliola/assertj-core/issues/355, so I just need to update documentation now.
